### DISCUSSION TITLE
Center v-card vertically and horizontally in Vuetify 3.0

### DISCUSSION THREAD

**Discussion by MrGonzo1024** [vuetifyjs/vuetify#17297] [Q&A] [answered]
Hi All,

Sorry for the newbie question and apologize if this has been asked before. I did try searching but didn't see an answer. I'm trying to center a v-card on the page both vertically and horizontally using Vuetify 3.0. "mx-auto" centers it horizontally but not vertically. I'm finding tons of examples out there for Vuetify 2.0 or lower, but none of them seem work with Vuetify 3.0. Is there a standard way to do it in Vuetify 3.0?

Thanks!

**Comment by yuwu9145** [comment, marked as answer]
What about using [flex utility classes](https://vuetifyjs.com/en/styles/flex/):

[demo](https://play.vuetifyjs.com/#eNqlUk1v2zAM/SusLtmA2A6anYJ06DAM2GH3HeoeFItJ2OmrEp1mCPLfR9kJFqQbUKA+2KZIvvfIp4eD+hJjvetRLdSS0UWrGT+3HmC5q3SMw+8QOE0eOqtzvmuVqdYW9/DUZ6b176pDz5hAW9r4U9CqU+vQ3OlkzuH5eSHDW8H6NJvFfauu01ukzZb/n2fcl+yPkNABxdw7MMGGBJkYtEOGLviMHSP3Is1QpEwd+Q2gJa7ha3AuGJpC0kzBIxhcEVOG515emBmsXgk47ILtI0vRqs838E0/9whdH8mQHCI48uT0FDSDLK9D+XJg7abg9MbTKApd6dqhJ+3AYwliSCmApRXKJ2ES/b03f+nQ3cgOl824vLMPEhYjRockGC1aNhfWSZi7RJEhy+glTU7IGA5Cs4YjrFNwMBHPJ6W4bInB5Q3clfyHyXe0NsDPkKy5mXws4COcAKmpGrFERKyfcvBybQ5FS3tK5FYtYDgpZ8JR4lZtmWNeNE1nvLQZcWCXao/c+OiaeylrUu+ZHFYmuPt5Pa9n1QpZ1/PGUObLbI3ZVasUXjImwWrV9IKt3MY3MJ4qhei2vh0ZTkeVmJ4LxSvoRvh2mKqE4pL49dbBrtr+MdxVxasBi4hj64+yfUv+V75afJeHpT+ctb5r8gGsID0WwuNUXcoVAUOfevwDuHRq2A==)

**Reply by MrGonzo1024** [reply]
Thank you so much @yuwu9145 ! It worked!  I have tried using **class="d-flex justify-center align-center"** before, but on v-sheet, v-container or div tags and none of those worked.  I guess it needed to be on v-main?  I see I need to learn more about the flex utility classes.

**Reply by yuwu9145** [reply]
It's nothing about v-main, flex utility classes is enough achieve it. You just need to make sure container element occupies full page height and width

**Reply by MrGonzo1024** [reply]
I see.  Thank you so much @yuwu9145
